template/1-overview.md

```diff
@@ -1,18 +1,9 @@
 # Overview
 
-*Maximum: 2 pages*
+Event Radar is an innovative app designed to effortlessly facilitate connections among individuals by streamlining event discovery, registration, and networking all into one place. It allows users to discover and register for upcoming events in their local area, enabling them to connect seamlessly with friends and new acquaintances they meet during the event. Event hosts can easily create and manage events, while also leveraging the app’s unique QR code system to scan tickets and enhance attendee engagement. By focusing on building a vibrant local community, Event Radar aims to revolutionize the way people interact and form connections in person.
```

Comment:
Here: "Event hosts can easily create and manage events, while also leveraging the app’s unique QR code system to scan tickets and enhance attendee engagement."
What do you mean by enhance attendee engagement?

Reply:
This is referencing the ability for hosts to send messages to event attendees. It is bad wording and can be changed.